package.json

```diff
@@ -30,7 +31,8 @@
     "husky": "^7.0.2",
     "lerna": "^3.22.1",
     "lint-staged": "^11.1.2",
-    "prettier": "^2.4.1"
+    "prettier": "^2.4.1",
+    "slice-machine-ui": "^0.4.2"
```

Comment:
why is slice-machine-ui now a dep?

Reply:
ah that's my mad it should not be. Think i accidentaly ran a command in the terminal, will remove